tests/product/tests/data/process/eks_process_test_cases.py

```diff
@@ -266,18 +276,116 @@
         **cis_eks_3_2_1_config_1,
         **cis_eks_3_2_2_config_1,
         **cis_eks_3_2_3_config_1,
-        **cis_eks_3_2_4_config_1,
-        **cis_eks_3_2_5_config_1,
+        **dict(
```

Comment:
same here, can we find an alternative for zipping and unpacking?

Reply:
Yes it is, we can modify `skip_param_case` function by changing function's signature. Change `skip_list` -> to dictionary, and change return value. So it will resolve complicated code with zip and unpack. Since this function used in all product functional tests, need to open a separate task for this.

Reply:
A new [task](https://github.com/elastic/security-team/issues/5758) was opened